Source/NSubstitute/Exceptions/CouldNotSetReturnException.cs

```diff
@@ -45,6 +45,17 @@ public class CouldNotSetReturnDueToNoLastCallException : CouldNotSetReturnExcept
 #if NET35 || NET4 || NET45
     [Serializable]
 #endif
+    public class CouldNotSetReturnDueToMissingInfoAboutPreviousCallException : CouldNotSetReturnException
+    {
+        public CouldNotSetReturnDueToMissingInfoAboutPreviousCallException() : base("Could not find information about the previous call to return from.") { }
```

Comment:
Fixed by renaming exception (and related tests) to use "Last" word instead of "Previous".
For stable scenario that of course doesn't bring value, but for some weird concurrency issues [like this one](https://github.com/nsubstitute/NSubstitute/issues/260) it's still better to have separate exceptions.

Let me know if you still want to change something after the latest adjustments.